deploy/helm/sumologic/templates/chart-configmap.yaml

```diff
@@ -3,6 +3,6 @@ kind: ConfigMap
 metadata:
   name: sumologic-configmap
 data:
-  fluentdLogs: {{ template "sumologic.metadata.name.logs" . }}
-  fluentdMetrics: {{ template "sumologic.metadata.name.metrics" . }}
+  fluentdLogs: {{ template "sumologic.metadata.name.logs.service" . }}
+  fluentdMetrics: {{ template "sumologic.metadata.name.metrics.service" . }}
   fluentdNamespace: {{ .Release.Namespace }}
```

Comment:
newline :scream:

Reply:
Added newline (and fixed editor settings to automatically add it next time).